psi4/src/export_functional.cc

```diff
@@ -213,25 +213,25 @@ void export_functional(py::module &m) {
                       std::shared_ptr<BasisExtents>>())
         .def("x",
              [](BlockOPoints &grid) {
-                 SharedVector ret(new Vector("X Grid points", grid.npoints()));
+                 auto ret = std::make_shared<Vector>("X Grid points", grid.npoints());
```

Comment:
Yay, this will save a lot of people time on their first explorations to find where on earth `SharedVector` and `SharedMatrix` are defined.

Reply:
Oh nice